test/WebJobs.Script.Tests.Integration/NodeEndToEndTests.cs

```diff
@@ -752,6 +753,7 @@ public void ErrorFunction_DoesNotBringDownHost()
             Assert.Equal(testData, (string)resultObject["reqBody"]);
             Assert.Equal("string", (string)resultObject["reqRawBodyType"]);
             Assert.Equal(testData, (string)resultObject["reqRawBody"]);
+            Assert.Equal("text/plain", resultObject["reqHeaders"]["content-type"]);
```

Comment:
Another great comment, all of our tests use `User-Agent`, which as mentioned in the other comment is its own special snowflake as it uses `" "` separator. I'll add some tests with the accept header which uses standard comma separators.